config/features/glance.yml

```diff
@@ -0,0 +1,15 @@
+features:
+  glance:
+    flags: ""
+    create_kvm_image:
```

Comment:
The coupling is actually intenational. If someone wants to reuse the config values within the same feature, he must create a new section in config. Within config this might be kept dry by using yaml anchors and references.